protected/humhub/modules/admin/views/group/index.php

```diff
@@ -36,7 +36,8 @@
                 'format' => 'html',
                 'value' => function (Group $group) {
                     return $group->name .
-                        ($group->is_default_group ? ' <span class="badge">' . Yii::t('AdminModule.user', 'Default') . '</span>' : '');
+                        ($group->is_default_group ? ' <span class="badge">' . Yii::t('AdminModule.user', 'Default') . '</span>' : '') .
```

Comment:
Maybe use the `humhub\widgets\Label` here.

Reply:
Ok, I have applied this in commit https://github.com/humhub/humhub/pull/4879/commits/87b0b4163c7863a41bcb27acca934d97deedf6cb but please note the `.label` is used there instead of `.badge`, for me it is almost same style, if we really need badge style I can revert the commit:

<img width="462" alt="label_vs_badge" src="https://user-images.githubusercontent.com/6995524/110089662-6c9a6400-7da7-11eb-94de-36787b5983d0.png">

Reply:
I think it looks better. We probably should get rid of `badges` in favor of `label`.